plugins/error.rst

```diff
@@ -1,4 +1,32 @@
 Error Plugin
 ============
 
-TODO: explain the error plugin
+Error plugin will transform response with specific status code into exception:
```

Comment:
i would say "The `ErrorPlugin` transforms responses with HTTP error status codes into exceptions:"